actors/miner/src/lib.rs

```diff
@@ -3418,10 +3421,11 @@ pub struct ReplicaUpdateInner {
 }
 
 enum ExtensionKind {
-    ExtendCommittmentLegacy, // handle only legacy sectors
-    ExtendCommittment,       // handle both Simple QAP and legacy sectors
-                             // TODO: when landing https://github.com/filecoin-project/builtin-actors/pull/518
-                             // ExtendProofValidity
+    ExtendCommittmentLegacy,
+    // handle only legacy sectors
+    ExtendCommittment, // handle both Simple QAP and legacy sectors
+                       // TODO: when landing https://github.com/filecoin-project/builtin-actors/pull/518
+                       // ExtendProofValidity
```

Comment:
Please move these comments around so that auto-format leaves them correctly attached.